### PR TITLE
Add super minimalistic query interface for nested data

### PR DESCRIPTION
**Pull request by peritus**

### CODE REVIEW

**Comment by peritus** [comment]
This depends on / includes commits in / is rebased on #8 .. so that should be merged first.

File: Dockerfile [hunk 1]
```diff
@@ -1,21 +1,20 @@
...
   postgresql-server-dev-all
 
-RUN pyvenv-3.4 --without-pip /opt/gather2-core-env/
-RUN curl https://bootstrap.pypa.io/get-pip.py | /opt/gather2-core-env/bin/python
-ADD ./gather2-core/requirements.txt /opt/gather2-core/requirements.txt
-RUN  /opt/gather2-core-env/bin/pip install -r  /opt/gather2-core/requirements.txt
-ADD ./gather2-core/ /opt/gather2-core/
+ADD ./requirements.txt /opt/tag/requirements.txt
+RUN pip install -r /opt/tag/requirements.txt
+ADD . /opt/tag
```
Comment (readevalprint) [inline Dockerfile]: Was this from a bad merge?

**Comment by readevalprint** [comment]
Are you sure this was merged correctly? it has the old folder scheme with `src` and stuff.

**Comment by peritus** [comment]
Rebased it again, now with #8 merged. Looks straight forward to me now. Thanks for noticing.

**Comment by readevalprint** [comment]
+1

**Pull request merged**